.gitpod.yml

```diff
@@ -13,7 +14,9 @@ vscode:
     - anweber.statusbar-commands@1.6.0
     - aaron-bond.better-comments
     - CoenraadS.bracket-pair-colorizer
-    - James-Yu.latex-workshop
+# latex workshop 8.20.2 on openvsx (released on august 24th) has a weird bug where the enter key stops working
+# https://github.com/James-Yu/LaTeX-Workshop/issues/2845
+    - James-Yu.latex-workshop@8.19.2
```

Comment:
wäre geil wenn das jemand behandeln würde. hier müssten wir entweder was gegen die auto-updates tun _oder_ auf https://github.com/open-vsx/publish-extensions/issues/414 warten

Reply:
bei neuen workspaces ist das kein problem.
Das ist nur ein Problem, wenn man in seinem VSCode ein aktuelleres installiert hat.

Reply:
da kann ich nichts zu sagen... das müsst ihr wissen

Reply:
unwichtig